src/ADAL.PCL.Desktop/PlatformInformation.cs

```diff
@@ -79,12 +79,27 @@ public override string GetOperatingSystem()
             return Environment.OSVersion.ToString();
         }
 
+        public override string GetDeviceId()
+        {
+            return DeviceID.strDeviceID;
```

Comment:
this will be null by default. consider using static initialization in DeviceID class.